ui/src/views/infra/UsageRecords.vue

```diff
@@ -0,0 +1,712 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <a-row style="margin-bottom: 5px;" :gutter="5">
+    <a-col :span="4">
+      <a-card
+        class="server-metrics"
+        :loading="serverMetricsLoading"
+      >
+        <a-statistic
+          :title="$t('label.server')"
+          :value="serverStats.hostname"
+        />
+      </a-card>
+    </a-col>
+    <a-col :span="4">
+      <a-card
+        class="server-metrics"
+        :loading="serverMetricsLoading"
+      >
+        <a-statistic
+          :title="$t('label.status')"
+          :value="serverStats.state"
+          :valueStyle="{ color: serverStats.state === 'UP' ? 'green' : 'red' }"
```

Comment:
Please check if there is any predefined values for color in config other wise good